.github/workflows/lint-test.yaml

```diff
@@ -36,12 +36,12 @@ jobs:
           fi
 
       - name: Run chart-testing (lint)
-        run: ct lint --config=ct.yaml
+        run: make lint
 
       - name: Create kind cluster
         uses: helm/kind-action@v1.8.0
         # Only build a kind cluster if there are chart changes to test.
         if: steps.list-changed.outputs.changed == 'true'
-
+        # TODO: make this test runnable from a makefile command
```

Comment:
this looks unrelated, please fork to another PR

Reply:
done